src/autoload/Global.gd

```diff
@@ -1,11 +1,63 @@
 extends Node
 
 
+const CONFIG_PATH := "user://config.cfg"
+
+var config := {
+	"name": "Guest",
+	"favorite_color": Color(255, 0, 0, 255),
```

Comment:
This will need to be removed/changed when @admoore0's changes are merged

Reply:
Worth wrapping the initial value into a constant? That way we always have it if the one in the variable gets weird or something. Your call.

Reply:
Not a bad idea, might make a `const default_config` which stores all defaults